code/modules/jobs/job_types/ncr.dm

```diff
@@ -384,14 +228,12 @@ Sergeant
 	display_order = JOB_DISPLAY_ORDER_SERGEANT
 	outfit = /datum/outfit/job/ncr/f13sergeant
 	exp_requirements = 600
-	matchmaking_allowed = list(
-		/datum/matchmaking_pref/mentor = NCR,
-	)
-	loadout_options = list(
-		/datum/outfit/loadout/sergeantrifleman,
-		/datum/outfit/loadout/sergeantrecon,
-		/datum/outfit/loadout/sergeantcqc,
-	)
+
+	loadout_options = list(	// ALL: Bayonet
+	/datum/outfit/loadout/sergeantrifleman,	// Service Rifle
+	/datum/outfit/loadout/sergeantrecon,	// M1/n Carbine, Binoculars, 9mm sidearm
+	/datum/outfit/loadout/sergeantcqc)	// Trench Shotgun, Gas mask, Smoke bombs, Trench knife
```

Comment:
```suggestion
	loadout_options = list( // ALL: Bayonet
		/datum/outfit/loadout/sergeantrifleman, // Service Rifle
		/datum/outfit/loadout/sergeantrecon, // M1/n Carbine, Binoculars, 9mm sidearm
		/datum/outfit/loadout/sergeantcqc, // Trench Shotgun, Gas mask, Smoke bombs, Trench knife
		)
```
Tabs will have different sizes depending on editor. A little space should suffice.